R/loadReg2.R

```diff
@@ -147,7 +148,8 @@ loadReg2 <- function(load.reg,
   }
   
   # Get the metadata, which we'll need to do the following data checks
-  meta <- getMetadata(load.reg)
+  #fill the site.id field, since rloadest only has a site.name equivalent
+  meta <- updateMetadata(getMetadata(load.reg), site.id = site.id)
```

Comment:
Re: your comment, "Wouldn't it fail right away if we take away the default, and you call it without supplying site.id?": Yep, that's true. Therefore, I'm proposing this change at this line. Instead of the above in line 154, do
```r
meta <- getMetadata(load.reg)
if(!missing(site.id)) {
  meta <- updateMetadata(meta, site.id = site.id)
}
```